definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField-nonprod.json

```diff
@@ -40,5 +40,19 @@
     "CaseFieldID": "generalApplicationNotApprovedReason",
     "UserRole": "caseworker-divorce-systemupdate",
     "CRUD": "CRUD"
+  },
+  {
+    "LiveFrom": "01/06/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "CaseFieldID": "generalApplicationOutcome",
+    "UserRole": "caseworker-divorce-financialremedy-judiciary",
+    "CRUD": "CRUD"
```

Comment:
you only need to have the case event toggled off

Reply:
Good call, ive moved these into the regular file